### ISSUE TITLE
Setting environment variables from CLI

### ISSUE THREAD

**Issue by martenjacobs** [resin-io/docs#805]
I'd like to add a number of service- and environment variables to my application from the CLI. It seems the current version of the documentation (or even the cli itself?) does not work properly with multicontainer set-ups. Using the docs and the current version of the cli, I'm able to set service variables for only one of my microservices and I'm completely unable to set environment variables.

**Comment by pimterry** [comment]
@martenjacobs yes, for multicontainer apps the CLI can't yet configure every kind of variable that you'll want. We're working on that, but for now for any features not yet fully supported you'll need to set those through the API directly.

See the [API overview](https://docs.resin.io/reference/api/overview/) for an introduction, and the [application config](https://docs.resin.io/reference/api/resources/application_config_variable/), [application environment](https://docs.resin.io/reference/api/resources/application_environment_variable/), and [service environment](https://docs.resin.io/reference/api/resources/service_environment_variable/) variable resources for details & examples.
